### ISSUE TITLE
iOS issue with V2/Tautulli

### ISSUE THREAD

**Issue by ghost** [Tautulli/Tautulli#1217]
**Version:**
2.0.13
**Branch:**
BETA
**Commit hash:**
e9725a00819cd103b1fd34141c581b65b340eed7
**Operating system:**
Ubuntu 16.04
**Python version:**
2.7.14
**What you did?**
Opened Tautulli on my iPhone to check current sessions
**What happened?**
Broken page
**What you expected?**
Not a broken page 
**How can we reproduce your issue?**
Start a session on Plex, open Tautulli. If you already have Tautulli open when the session starts it looks normal. This did not happen on V1/PlexPy.
**What are your (relevant) settings?**
**Link to logs:**

How it looks like when nothing is playing (normal) https://i.imgur.com/E5tnPiG.png
If you open Tautulli when a stream is already running (BROKEN) https://i.imgur.com/EbYRorO.png
If you have Tautulli open when a stream starts (normal) https://i.imgur.com/HF2qDpy.png

Sorry for asking this here but FeatHub is down. Please add an option to disable Terminate Stream in V2.

**Comment by wcomartin** [comment]
Hi, Which iPhone/iOS version are you running on?

**Comment by ghost** [comment]
iPhone 7, iOS 11.2.2

**Comment by ghost** [comment]
I have this same issue on an 10.5” iPad Pro running 11.2.2. If I see the broken page, I can go to another tab (Libraries or Users) and back to Home, and it will look fine.

**Comment by ghost** [comment]
That actually does work, but to clarify, you need to hit the back button on your browser app. Going into "Users" and then using the burger menu to press home icon still gives you a broken page.

Should get me by for now though. I was actually running both V1/V2 so that I can test the new version but still access it from my iPhone.

**Comment by sn00pster** [comment]
I posted about this issue on both the reddit and Plex forums threads, I included a video showing the behaviour there.

Works for me on my iPhone 7+ but is broken on my iPad Pro 10.5

Can confirm as above that if you're already on the page and start a stream it shows correctly, but if you refresh you get the broken screen.  The "fix" above of clicking users and then using the back button works, but you cannot hit the home button because this gives a broken page.
